dev/MRTCore/global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "5.0.100-rc.2.20480.7"
+        "version": "5.0.100-rtm"
```

Comment:
>5.0.100-rtm [](start = 20, length = 11)

if this is the new thing, we may need to update the build pipeline too

Reply:
Yes, we should update the build pipeline. WinUI preview 3 requires .net 5, so this change is necessary. .net5 was released a week ago.

Reply:
hmm, don't see your comments in CF at all. Switch to web version.  Should change mrtcore\versions.props